src/components/CardConfig/index.tsx

```diff
@@ -13,7 +13,11 @@ export const CardConfig: FC = () => {
           <CardConfigHeader />
         </ExpansionPanel.Header>
         <ExpansionPanel.Content>
-          <CardConfigContent className="mt-3" />
+          <CardConfigContent
+            className="mt-3"
+            concurrency={1}
+            interval={'50ms'}
```

Comment:
For string you don't need to wrap with bracket you can write :
`interval='50ms'`

Reply:
Done